ams/api/src/main/java/com/netease/arctic/ams/api/client/ArcticThriftUrl.java

```diff
@@ -45,6 +48,14 @@ public class ArcticThriftUrl {
   public static final int maxRetries = 3;
   private static final Logger logger = LoggerFactory.getLogger(ArcticThriftUrl.class);
   private static final Pattern PATTERN = Pattern.compile("zookeeper://(\\S+)/([\\w-]+)");
+
+  private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
```

Comment:
Can we move the `JacksonUtils` implementation into the `ams-api` module? So we can use it directly here.

Reply:
Seems we use JacksonUtils more in `ams-server` module, and only one usage in `ams-api`, and `ams-server` does not dependent on `ams-api`, so didn't create it in `ams-api`
I'll try to add dependent `ams-api` in `ams-server` and move `JacksonUtls`